db.go

```diff
@@ -853,3 +853,10 @@ func (x *dummyPermitDB) SetPermit(userId UserId, permit *Permit) error {
 
 func (x *dummyPermitDB) Close() {
 }
+
+func (a *AuthUser) getIdentity() string {
+	if (&a.Email == nil || len(a.Email) == 0) {
```

Comment:
I've seen it done this way in other go code - is it safe to assume the address will never be nil?